### ISSUE TITLE
detection of CNV in germlines

### ISSUE THREAD

**Issue by lmanchon** [diazlab/phyngle#37]
--Hi,

2 questions:
is CONICS able to detect CNV in germlines single cells ?
and is there a way to compute mosaicism rate ?

thank you --

**Comment by diazlab** [comment]
Yes, but you will need to define a control population without CNVs.

You will get a CNV call for each cell which you could use to estimate the percentage of cells with the alteration.
